docs/developer-docs/integrations/sns/integrate-sns/index-integration.md

```diff
@@ -0,0 +1,56 @@
+# SNS Index Canister
+
+The index canister fetches transactions from the [ledger canister](ledger-integration.md) and indexes them by Account. 
+It allows to query the transactions of an Account in descending order and the list of Account that belongs to a Principal. 
```

Comment:
is it clear what is descending? I assume the transaction ID, right? 
Also, if query for the transaction of an account don't I just get back the transaction IDs (rather than the full transactions)? If so, I think this might not be fully clear from this description.

Reply:
The returned transaction is containing not only the id but also the details (depending on the type if it's a mint, transfer, etc.). Will clarify it.

Reply:
updating to clarify